## Welcome to GitHub Pages

You can use the [editor on GitHub](https://github.com/HaihuaWang-hub/haihua.github.io/edit/gh-pages/index.md) to maintain and preview the content for your website in Markdown files.

Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.

### Markdown

Markdown is a lightweight and easy-to-use syntax for styling your writing. It includes conventions for

```markdown
Syntax highlighted code block

# Header 1
## Header 2
### Header 3

- Bulleted
- List

1. Numbered
2. List

**Bold** and _Italic_ and `Code` text

[Link](url) and ![Image](src)
```

For more details see [Basic writing and formatting syntax](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

### Jekyll Themes

Your Pages site will use the layout and styles from the Jekyll theme you have selected in your [repository settings](https://github.com/HaihuaWang-hub/haihua.github.io/settings/pages). The name of this theme is saved in the Jekyll `_config.yml` configuration file.

### Support or Contact

Having trouble with Pages? Check out our [documentation](https://docs.github.com/categories/github-pages-basics/) or [contact support](https://support.github.com/contact) and we’ll help you sort it out.




Skip to content
Search or jump to…
Pull requests
Issues
Marketplace
Explore
 
@HaihuaWang-hub 
data-science-sequencing
/
data-science-sequencing.github.io
Public
11
53
23
Code
Issues
1
Pull requests
Actions
Projects
Wiki
Security
Insights
data-science-sequencing.github.io/index.html
@jessemzhang
jessemzhang announcement for assignment solutions being up
Latest commit fd9c784 on Mar 19, 2018
 History
 3 contributors
@jessemzhang@govinda-kamath@dntse
113 lines (100 sloc)  7.31 KB
   
---
layout: default
---

<div>
  This website accompanies the course <b>EE 372: Data Science for High-Throughput Sequencing</b>.  <br>
  For questions/comments/typos in the course notes please leave a comment in the notes, submit a pull request
  directly to our <a href="https://github.com/data-science-sequencing/data-science-sequencing.github.io">Git repo</a>, or email us. Click <a href="/Spr2016/Spr2016/">here</a> for last offering's course website.
</div>

<div class ="announcements">
  <div class="module-header">Announcements</div>
  <div class="materials-bullets-notopmargin">
  <ul>
  <li>19 March 2018: All assignment solutions now posted and accessible from the assignment pages.</li>
  <li>14 March 2018: <a href="/Win2018/project/abstracts">Project abstracts</a> posted.</li>
  <li>12 March 2018: <a href="/Win2018/assignments/assignment3/">Assignment 3</a> deadline extended to Friday 16 March 2018 at 11:59pm. </li>
  <li>28 February 2018: <a href="/Win2018/assignments/assignment3/">Assignment 3</a> released. Due on Wednesday 14 March 2018 at 11:59pm. Submission through <a href="https://gradescope.com/">Gradescope</a>. </li>
  <li>6 February 2018: <a href="/Win2018/assignments/assignment2/">Assignment 2</a> released. Due on Tuesday 20 February 2018 at 11:59pm. Submission through <a href="https://gradescope.com/">Gradescope</a>. </li>
  <li>29 January 2018: <a href="/Win2018/project/guidelines">Project guidelines</a> handout posted. Please sign up for an office hour slot <a href="https://docs.google.com/spreadsheets/d/1ySZWc7qNUYyR0Ayj3adITypp-Ea2u8L9vF9aRAEKbv8/edit?usp=sharing">here</a>.</li>
  <li>17 January 2018: David's office hour on Thursday 18 January will be changed to Friday 19 January from 3:00-4:00pm. Govinda and Jesse will be holding office hours from 4:00-5:00pm Friday 19 January on the 3rd floor of Packard (kitchen area). </li>
  <li>17 January 2018: Please fill out this <a href="https://docs.google.com/spreadsheets/d/1r318qvy0F6io8onBz1JTAgjgTW7fg4spMF5iObLDSJk/edit?usp=sharing">Google doc</a> for final project groups. </li>
  <li>17 January 2018: <a href="/Win2018/assignments/assignment1/">Assignment 1</a> released. Due on  Friday 26 January 2018 at 11:59pm. Submission through <a href="https://gradescope.com/">Gradescope</a> (entry code: M5V4JJ). </li>
  <li>9 January 2018: <a href="/Win2018/handouts/Overview.pdf">Course Description</a> handout posted.</li>
  <li>8 January 2018: <a href="/Win2018/lectures/lecture0/">Course Outline</a> posted.</li>
  </ul>
</div>

<div class ="course-description">

   <div class ="module-header">Course Description</div>
   Extraordinary advances in sequencing technology in the past decade have revolutionized biology and medicine. Many high-throughput sequencing based assays have been designed to make various biological measurements of interest. This course explores the various computational and statistical problems that arises from processing high throughput sequencing data. Specific problems we will study include genome assembly, haplotype phasing, RNA-Seq quantification, single cell RNA-seq analysis, etc.  Specific techniques we will learn to solve these problems include spectral algorithms, dynamic programming, the EM algorithm, PCA, FDR, etc. Through this course, the student will also get familiar with various software tools developed for the analysis of real sequencing data.

  <div class ="course-staff">
     <div class="module-subheader">Course Staff</div>
     Instructor: <a href="https://web.stanford.edu/~dntse/"> David Tse </a> (dntse _at_ stanford.edu) <br>
     Teaching assistants: <a href="https://web.stanford.edu/~gkamath"> Govinda Kamath </a> (gkamath _at_ stanford.edu) , <a href="https://web.stanford.edu/~jessez"> Jesse Zhang </a> (jessez _at_ stanford.edu) <br>
     Office hours: Mon 3:00-4:00pm and Thurs 3:15-4:15pm at Packard 264 for instructor, Mon 11:00am-12:00pm at Packard 104 for teaching assistants
  </div>

  <div class ="Lecture-times">
     <div class="module-subheader">Lectures times</div>
      Tuesday, Thursday 1:30-2:50pm at 540-108 <br>
  </div>

  <div class="grading">
    <div class="module-subheader">Grading</div>
    <div class="materials-bullets-notopmargin">
    <ul>
    <li>Class participation: 10%</li>
    <li>Scribing: 10%</li>
    <li>Problem sets (3-4) : 30%</li>
    <li>Project: 50%</li>
    </ul>
  </div>

  <div class="reference-wrap">
    <div class="module-subheader">Useful References</div>
        <a href="http://www.langmead-lab.org/teaching-materials/">Ben Langmead's lecture notes</a><br>
        <a href="http://bioinformaticsalgorithms.com/index.htm"> Bioinformatics algorithms by Compeau and Pevzner</a>
  </div>
</div>



<div class="home">
  <div class="materials-wrap">
    <div class="module-header">Course Materials</div>
    <div class="materials-item">
        <a href="/Win2018/lectures/lecture0/">Course Outline</a>
        <a href="/Win2018/lectures/lecture1/">Lecture 1: Introduction</a>
        <a href="/Win2018/lectures/lecture2/">Lecture 2: Basics of DNA & Sequencing by Synthesis</a>
        <a href="/Win2018/lectures/lecture3/">Lecture 3: Base Calling for Second-Generation Sequencing</a>
        <a href="/Win2018/lectures/lecture4/">Lecture 4: Base Calling for Next-Generation Sequencing</a>
        <a href="/Win2018/lectures/lecture5/">Lecture 5: Assembly - An Introduction</a>
        <a href="/Win2018/lectures/lecture6/">Lecture 6: Assembly - Greedy Algorithm</a>
        <a href="/Win2018/lectures/lecture7/">Lecture 7: Assembly - De Bruijn Graph</a>
        <a href="/Win2018/lectures/lecture8/">Lecture 8: Assembly - Towards a Long Reads Assembler</a>
        <a href="/Win2018/lectures/lecture9/">Lecture 9: Alignment - Dynamic Programming and Indexing</a>
        <a href="/Win2018/lectures/lecture10/">Lecture 10: Haplotype Phasing - Community Recovery</a>
        <a href="/Win2018/lectures/lecture11/">Lecture 11: Haplotype Phasing - Spectral Stitching</a>
        <a href="/Win2018/lectures/lecture12/">Lecture 12: RNA-seq - A Counting Problem</a>
        <a href="/Win2018/lectures/lecture13/">Lecture 13: RNA-seq - Quantification and the EM Algorithm Part 1</a>
        <a href="/Win2018/lectures/lecture14/">Lecture 14: RNA-seq - Quantification and the EM Algorithm Part 2</a>
        <a href="/Win2018/lectures/lecture15/">Lecture 15: Differential Analysis and Multiple Testing</a>
        <a href="/Win2018/lectures/lecture16/">Lecture 16: Single Cell RNA-Seq - Introduction</a>
        <a href="/Win2018/lectures/lecture17/">Lecture 17: Multiple Testing</a>
        <a href="/Win2018/lectures/lecture18/">Lecture 18: Empirical Bayes and Single-Cell RNA-Seq Analysis</a>
        <a href="/Win2018/lectures/lecture19/">Lecture 19: Single-Cell RNA-Seq - Clustering</a>
    </div>
  </div>
</div>

<div class="home">
  <div class="assignments-wrap">
    <div class="module-header">Assignments</div>
    <div class="materials-item">
        <a href="/Win2018/assignments/assignment1/">Assignment 1</a>
        <a href="/Win2018/assignments/assignment2/">Assignment 2</a>
        <a href="/Win2018/assignments/assignment3/">Assignment 3</a>
    </div>
  </div>
</div>

<div class="home">
  <div class="assignments-wrap">
    <div class="module-header">Project</div>
    <div class="materials-item">
    <a href="/Win2018/project/guidelines">Project Guidelines and Ideas</a>
    <a href="/Win2018/project/abstracts">Project Abstracts</a>
    </div>
  </div>
</div>
© 2021 GitHub, Inc.
Terms
Privacy
Security
Status
Docs
Contact GitHub
Pricing
API
Training
Blog
About
Loading complete
